controllers/suite_test.go

```diff
@@ -74,16 +81,31 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	done := make(chan interface{})
-	go func() {
 
+	// write logs from reconciler to a log file
+	var logFile io.Writer
+	logFile, err := os.Create("/tmp/ocs-osd-deployer.log")
+	Expect(err).ToNot(HaveOccurred())
+
+	go func(logFile io.Writer) {
+
+		GinkgoWriter.TeeTo(logFile)
 		logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
+		ctx, cancel = context.WithCancel(context.TODO())
```

Comment:
Please change to 
```suggestion
		ctx, cancel = context.WithCancel(context.Background())
```

For a full explanation you can take a look here:
https://pkg.go.dev/context#Background